Source/hydro/trace_ppm.F90

```diff
@@ -522,31 +514,16 @@ subroutine trace_ppm_rhoe(lo, hi, &
                 dup = un_ref - Im(3,QUN) - hdt*Im_src(3,QUN)
                 dptotp = p_ref - Im(3,QPRES) - hdt*Im_src(3,QPRES)
 
-
-                ! Optionally use the reference state in evaluating the
-                ! eigenvectors
-                if (ppm_reference_eigenvectors == 0) then
-                   rho_ev  = rho
-                   cc_ev   = cc
-                   csq_ev  = csq
-                   h_g_ev = h_g
-                else
-                   rho_ev  = rho_ref
-                   cc_ev   = cc_ref
-                   csq_ev  = csq_ref
-                   h_g_ev = h_g_ref
-                endif
-
                 ! (rho, u, p, (rho e) eigensystem
 
                 ! These are analogous to the beta's from the original PPM
                 ! paper (except we work with rho instead of tau).  This is
                 ! simply (l . dq), where dq = qref - I(q)
 
-                alpham = HALF*(dptotm*(ONE/(rho_ev*cc_ev)) - dum)*(rho_ev/cc_ev)
-                alphap = HALF*(dptotp*(ONE/(rho_ev*cc_ev)) + dup)*(rho_ev/cc_ev)
-                alpha0r = drho - dptot/csq_ev
-                alpha0e_g = drhoe_g - dptot*h_g_ev  ! note h_g has a 1/c**2 in it
+                alpham = HALF*(dptotm*rho_ref_inv*cc_ref_inv - dum)*rho_ref*cc_ref_inv
+                alphap = HALF*(dptotp*rho_ref_inv*cc_ref_inv + dup)*rho_ref*cc_ref_inv
+                alpha0r = drho - dptot/csq_ref
```

Comment:
Since we're doing this divide by csq_ref twice now, should it have its own inverse variable too?